src/widgets/_entries.scss

```diff
@@ -14,6 +14,10 @@ entry {
         color: #{'@placeholder_text_color'};
     }
 
+    &:selected {
```

Comment:
Can you give an example of when an entry would be selected?

Reply:
By default a Gtk.Entry is selected unless you remove it's focus to another widget at creation, right? This is where this comes into play.

Reply:
You're thinking of :focus